client/src/components/EditProfile/EditProfileTab/EditProfileTab.tsx

```diff
@@ -0,0 +1,320 @@
+    if (accountType === 'partner') {
+      alert(`
+        available: ${available}\n
+        availability: ${availability}\n
+        first name: ${firstName}\n
+        last name: ${lastName}\n
+        gender: ${gender}\n
+        birthdate: ${birthdateMonth} ${birthdateDay}, ${birthdateYear}\n
+        email: ${email}\n
+        phone: ${phone}\n
+        address: ${address}\n
+        description: ${description}
+      `);
+    } else {
+      alert(`
+        first name: ${firstName}\n
+        last name: ${lastName}\n
+        gender: ${gender}\n
+        birthdate: ${birthdateMonth} ${birthdateDay}, ${birthdateYear}\n
+        email: ${email}\n
+        phone: ${phone}\n
+        address: ${address}\n
+        description: ${description}
+      `);
+    }
+  };
+
+  return (
+    <Box>
+      <Typography className={classes.title}>Edit Profile</Typography>
+      <form className={classes.form} onSubmit={handleSubmit}>
+        {accountType === 'partner' && (
+          <Box className={classes.section}>
+            <FormLabel>
+              <Typography className={classes.label}>i&apos;m available</Typography>
+            </FormLabel>
+            <Switch
+              checked={available}
+              onChange={({ target: { value } }) => setAvailable(!available)}
+              color="primary"
+            />
+          </Box>
+        )}
+
+        {accountType === 'partner' && (
+          <Box className={classes.section}>
+            <FormLabel>
+              <Typography className={classes.label}>availability</Typography>
+            </FormLabel>
+            <Select
+              className={classes.selectAvailability}
+              id="availability"
+              value={availability}
+              onChange={({ target: { value } }) => setAvailability(value)}
+              variant="outlined"
+              style={availability !== 'availability' ? { color: 'black' } : { color: 'grey' }}
+            >
+              <MenuItem value="availability" disabled>
+                availability
+              </MenuItem>
+              <MenuItem value="<10">Less than 10 hrs/week</MenuItem>
+              <MenuItem value="10-20">10-20 hrs/week</MenuItem>
+              <MenuItem value="20-30">20-30 hrs/week</MenuItem>
+              <MenuItem value=">30">More than 30 hrs/week</MenuItem>
+            </Select>
+          </Box>
+        )}
+
+        <Box className={classes.section}>
+          <FormLabel>
+            <Typography className={classes.label}>first name</Typography>
+          </FormLabel>
+          <OutlinedInput
+            className={classes.input}
+            id="firstName"
+            placeholder="John"
+            name="firstName"
+            autoComplete="given-name"
+            autoFocus={accountType === 'client'}
+            value={firstName}
+            onChange={({ target: { value } }) => setFirstName(value)}
+          ></OutlinedInput>
+        </Box>
+        <Box className={classes.section}>
```

Comment:
Optional: I think we can componentize this block of code for increased readability and reusability :)